citest/google_session/google_session.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+
+from bs4 import BeautifulSoup
```

Comment:
I'm not against adding useful external packages, but am hesitant to add them for trivial reasons. All you use from bs4 is BeautifulSoup.find(). I dont have an appreciation for how complex that is, but is it interesting enough to justify? Are there other simple implementations using standard packages? Again, I'm not against adding it if it provides real value. But if it is just a small convenience then I personally prefer a bit more code and tests over increased dependencies (and their potential transitive closure).

Of course, it could be that adding this package would let us throw out some other custom implementations or be useful later. But if it is just speculative, I'd rather defer now and refactor later.